test/provider/AppDirectory-get.unittest.ts

```diff
@@ -0,0 +1,474 @@
+import 'jest';
```

Comment:
We have a convention in the integration tests of putting "categoryies" of tests under a subdirectory. I think it would make sense to do that here too, i.e. `/provider/AppDirectory/get.unittest.ts`.

Reply:
I'd wanted with the unit tests to match the structure of the files under test (same as in Notifications) - I've moved the unit tests to actually do that.

Open to still grouping by folder, but would prefer to keep it that for unit tests, folders are there to match the structure we're mirroring

Reply:
Ok, that's fine. Forgot that's how we were structuring for the unit tests.